src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -471,6 +473,44 @@ public String getQuestionResultStatisticsCsv(
         }
         return optionPoints;
     }
+    
+    /**
+     * Used to update the option points mapping with a default value when distributing to recipients
+     * @param optionPoints
+     * @param question question for the option points mapping
+     * @param responses responses for the option points mapping
+     * @param bundle
+     */
+    private void updateRecipientOptionPointsMappingWithDefaultValue(
+            Map<String, List<Integer>> optionPoints,
+            FeedbackQuestionAttributes question,
+            List<FeedbackResponseAttributes> responses,
+            FeedbackSessionResultsBundle bundle) {
+        
+        if (!distributeToRecipients) {
```

Comment:
Might be more readable if this check was moved to a higher level, right before this method gets called. The control flow should be more obvious and you get to shorten the method name. But feel free to leave this as it is.